polybar-scripts/timezone-switcher/README.md

```diff
@@ -0,0 +1,27 @@
+# Timezone Switcher
```

Comment:
Please use `info-timezone` as script name and rename all files and the directory.